test/check-machines-settings

```diff
@@ -644,8 +644,8 @@ class TestMachinesSettings(VirtualMachinesCase):
         # Check rebooting machine will not unexpectedly affect watchdog configuration
         setWatchdogActionLive(action="poweroff", reboot_machine=True)
 
-        # Older libvirt in rhel-8-9 and centos-8-stream doesn't support target.hotplug=off
-        if not m.image.startswith("rhel-8") and m.image != "centos-8-stream":
+        # Older libvirt doesn't support target.hotplug=off
+        if not m.image.startswith("rhel-8") and m.image not in ["centos-8-stream", "debian-stable"]:
             # Disable hotplugging for subVmTest1
             m.execute("virsh destroy subVmTest1")
             m.execute("virt-xml subVmTest1 --edit model=pci-root --controller target.hotplug=off")
```

Comment:
On the second thought, I just realized maybe we should have something that tells us that this condition is no longer needed. With rhel-8 and centos-8-stream that was simple, we would remove the condition once we will not support those versions. But since "debian-stable" does not contain version in its name, than here is a chance that the condition will linger forever.
So maybe we should also add condition to check that version of libvirt is indeed too log. Accoding to [libvirt notes](https://libvirt.org/news.html), the target.hotplug functionality was added at 7.9.0.:

```suggestion
        # Older libvirt doesn't support target.hotplug=off
        if m.image.startswith("rhel-8") or m.image in ["centos-8-stream", "debian-stable"]:
            self.assertLess(m.execute("virsh --version"), "7.9.0")
        else:
            # Disable hotplugging for subVmTest1
            m.execute("virsh destroy subVmTest1")
            m.execute("virt-xml subVmTest1 --edit model=pci-root --controller target.hotplug=off")
```